lua/tardis/cl_hud.lua

```diff
@@ -0,0 +1,20 @@
+//TARDIS HUD
+
+function TARDISHUD()
+    if not TARDIS:GetSetting("health-enabled", true) then return end
+    if not (LocalPlayer():GetTardisData("interior") or LocalPlayer():GetTardisData("exterior")) then return end
+    local tardis = LocalPlayer():GetTardisData("exterior")
+    if not IsValid(tardis) then return end
+    local health = tardis:GetData("health-val", 0)
+    local minwidth = 160
+    local width = minwidth + math.Clamp( ((#tostring(health)-4) * 30), 0, math.huge)
+    local height = 120
+    local x = (ScrW()-width)*0.02
+    local y = (ScrH()-height)*0.025
+    draw.RoundedBox( 10, x, y, width, height, NamedColor("BgColor") )
+    local textcolor = (health > 0) and NamedColor("FgColor") or NamedColor("Caution")
+    draw.DrawText( "TARDIS", "TARDIS-PageName", x+10, y+10, textcolor, TEXT_ALIGN_LEFT )
+    draw.DrawText( health, "TARDIS-Large", x+10, y+45, textcolor, TEXT_ALIGN_LEFT )
```

Comment:
Are these two fonts defined anywhere? Most likely in legacy code but may need copying to this addon we don't want legacy to be a dependency

Reply:
Those are defined in the Rewrite itself I believe, so there should be no need

Reply:
Okay just make sure they are then we're good

Reply:
![image](https://user-images.githubusercontent.com/7284418/81522405-dc26fa80-9320-11ea-9ba4-cc0237d579cc.png)
lua/tardis/sh_screens.lua 22-45